tools/hive/hadoop-2.6.0/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
Is this license file required? We don't seem to be including any of the hadoop source here.